### DISCUSSION TITLE
MaximoSystemLibrary not found issue resolution

### DISCUSSION THREAD

**Discussion by cehlgngh** [eclipse-birt/birt#1126] [General]
When configuring BIRT Eclipse for Maximo, the IDE reported the following error amongst others:

The library with the namespace "MaximoSystemLibrary" is not found.

This error was resolved by the following steps:

Login to the Maximo front end and export the MaximoSystemLibrary.rptlibrary from the Report Administration application. In the location where the BIRT project is configured to find this file, overwrite the same file with the new exported file.  Refreshing the library, project, closing/reopening the report and restarting Eclipse after placing the new file resolved the issue.

**Comment by bahdabooom** [comment]
Thank you for this resolution!

**Comment by vishwam7** [comment]
That worked like a charm, Thanks for the solution! For some reason BIRT system felt like the hardest thing I ever dealt with. This solution is also applicable for MAS8 and higher and birt_480 and higher.
